release-22.2.0: backupccl: upgrade restoreTPCCIncLatest/nodes=10 backup fixture

msbutler opened the pull request:
Backport 1/1 commits from #90669.

/cc @cockroachdb/release

---

restoreTPCCIncLatest/nodes=10 has been consistently failing because the backup fixture is more than one major version behind the cluster running the restore. This patch updates the backup fixture used by this roachtest.

Informs #89380

Release note: None

Release justification: test only change to fix a failing roachtest

msbutler committed:
backupccl: deflake roachtest.restore2TB/nodes=10/with-pause

This test pauses a restore several times, but would fail if it attempted to
pause the job after the job already succeeded. This patch prevents the test
from failing if this occurs.

Informs #90023

Release note: None

msbutler committed:
backupccl: upgrade restoreTPCCIncLatest/nodes=10 backup fixture

restoreTPCCIncLatest/nodes=10 has been consistently failing because the backup
fixture is more than one major version behind the cluster running the
restore. This patch updates the backup fixture used by this roachtest. Note
that this fixture was created on 22.2.0-alpha.4, which means that on master,
we're technically doing a "cross version" restore. We decided not to re-create
a fixture for master because a backup taken from master will change
significantly over the release cycle. Once a release SHA is decided for 22.2.0,
the fixtures for master will be updated again.

This patch also updates the tpccInc harness to use the new fixture.

Informs #88090 #89380

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/90857)
<!-- Reviewable:end -->

msbutler committed:
backupccl: ensure restore2TB/nodes=10/with-pause pauses at least once

This patch adds a check that will cause the roachtest to fail if the job was
not paused at least once.

Release note: None

Epic: none

msbutler commented:
@adityamaru making sure you're aware of this pr as well. it's identical to the 22.2 backport.

adityamaru commented:
@msbutler LGTM but 22.2.0 needs a TL stamp so will have to go through release-backports.
